### DISCUSSION TITLE
Package minted Error: You must invoke LaTeX with the -shell-escape flag.

### DISCUSSION THREAD

**Discussion by changshengyi** [MiKTeX/miktex#767] [Q&A]
```sh
Package ifplatform Warning: 
    shell escape is disabled, so I can only detect \ifwindows.

) (C:\Program Files\MiKTeX\tex/latex/framed\framed.sty)
(C:\Program Files\MiKTeX\tex/latex/newfloat\newfloat.sty))

! Package minted Error: You must invoke LaTeX with the -shell-escape flag.

See the minted package documentation for explanation.
Type  H <return>  for immediate help.
 ...                                  
```

When i use minted package, i got this error. I know how to fix it in command, but i do not know how to add `-shell-escape` parameter in TeXworks of MiKTeX

**Comment by dreckshippy** [comment]
You can configure the commands TeXworks uses by going to Edit -> Preferences -> Typesetting. Then under "Processing Tools" mark pdfLaTeX (or which ever program you use) and hit "Edit..." on the right.
Now press the small + next to the Arguments box, enter the argument you want to add i.e. `-shell-escape`. Then you have to move this before the `$fullname` argument, as it needs to be processed as a flag, which are given to pdflatex before the filename.

**Reply by spamegg1** [reply]
This gives:
```
-shell-escape: unknown option

Sorry, but texify did not succeed.
```
